### PR TITLE
Add aspect `aspectsYEAH`

### PR DESCRIPTION
**Pull request by Techievena**
Add aspects for the aspectsYEAHBear

Closes https://github.com/coala/coala/issues/4040

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/bearlib/aspects/Spelling.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+@Spelling.subaspect
+class aspectsYEAH:
+    """
+    This aspect dictates that the term ``aspects`` and ``aspect`` must begin
+    with lower-case letters.
+    """
+    class docs:
+        example = """
+        # Aspects are the mother of all futures of coala
```
Comment (userzimmermann) [inline coalib/bearlib/aspects/Spelling.py]: Should the example be a wrong case? I'm not sure...

File: coalib/bearlib/aspects/Spelling.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    """
+    This aspect describes spelling of your source code.
+    """
+
+
+@Spelling.subaspect
+class aspectsYEAH:
+    """
+    This aspect dictates that the term ``aspects`` and ``aspect`` must begin
+    with lower-case letters.
```
Comment (userzimmermann) [inline coalib/bearlib/aspects/Spelling.py]: Maybe even point out that both terms must be **all lowercase**

File: coalib/bearlib/aspects/Spelling.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    """
+    class docs:
+        example = """
+        # Aspects are the mother of all futures of coala
+        """
+        example_language = 'All'
+        importance_reason = """
+        aspects are fundamental concepts in coala and therefore have to begin
+        with lower-case letters.
```
Comment (userzimmermann) [inline coalib/bearlib/aspects/Spelling.py]: What about *"are too fundamental to contain any upper case letters"*? :D

File: coalib/bearlib/aspects/Spelling.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        """
+        fix_suggestions = """
+        ``aspects`` or ``aspect`` are always written with a lower case ``a``.
```
Comment (userzimmermann) [inline coalib/bearlib/aspects/Spelling.py]: All lowercase :)

File: coalib/bearlib/aspects/Spelling.py (hunk 2 above)
Comment (Asnelchristian) [inline coalib/bearlib/aspects/Spelling.py]: and ``aspectsYEAH`` :p

**Comment by userzimmermann** [comment]
Perfect! Let's go on like this...

**#aspectsYEAH**

**Comment by userzimmermann** [comment]
ack https://github.com/coala/coala/pull/4049/commits/744fc56572c36c318af7e0da18e1b1d64c62afa6

**Comment by userzimmermann** [comment]
@rultor merge

**Pull request merged**